Resources/translations/validators.en.yml

```diff
@@ -1 +1,2 @@
 craueFormFlow.previousStepInvalid: The form for step %stepNumber% is invalid. Please go back and try to submit it again.
+craueFormFlow.formExpired: This form has expired. Please resubmit your information.

```

Comment:
Hmm. I'm going to suggest that we keep this as "This form has expired." This is a message intended for end-users, and I think they understand things as "forms" not "flows", even though as programmers we're making a distinction internally between a single form page and a flow of form pages. I don't think the typical user would understand what "flow" means.